src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Windows_Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{5EE76DCC-9FD5-47FD-AB45-BD0F0857740F}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>System.Net.Requests</RootNamespace>
+    <AssemblyName>System.Net.Requests</AssemblyName>
+    <AssemblyVersion>4.0.10.0</AssemblyVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <!-- Help VS understand available configurations -->
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FreeBSD_Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FreeBSD_Release|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Linux_Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Linux_Release|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU' " />
+
+  <ItemGroup>
+    <Compile Include="System\Net\HttpRequestHeader.cs" />
+    <Compile Include="System\Net\HttpWebRequest.cs" />
+    <Compile Include="System\Net\HttpWebResponse.cs" />
+    <Compile Include="System\Net\IWebRequestCreate.cs" />
+    <Compile Include="System\Net\ProtocolViolationException.cs" />
+    <Compile Include="System\Net\RequestStream.cs" />
+    <Compile Include="System\Net\SystemWebProxy.cs" />
+    <Compile Include="System\Net\TaskExtensions.cs" />
+    <Compile Include="System\Net\WebException.cs" />
+    <Compile Include="System\Net\WebExceptionStatus.cs" />
+    <Compile Include="System\Net\WebHeaderCollection.cs" />
+    <Compile Include="System\Net\WebRequest.cs" />
+    <Compile Include="System\Net\WebResponse.cs" />
+
+    <Compile Include="$(CommonPath)\Interop\Windows\wininet\Interop.wininet_errors.cs">
+      <Link>Common\Interop\Windows\wininet\Interop.wininet_errors.cs</Link>
+    </Compile>
```

Comment:
This is temporary for now.  Later the `HttpRequestException` object in the System.Net.Http library will change.  It will get a `Status` property that is an enum type similar to the old .NET Desktop `WebErrorStatus`.  That enum will be platform agnostic.  So, thus System.Net.Requests which builds on top of System.Net.Http will surface that enum value as well.

Any platform-specific mapping of errors will be contained in the System.Net.Http library.